src/ReactPixiFiber.js

```diff
@@ -142,12 +165,45 @@ const ReactPixiFiber = ReactFiberReconciler({
         );
         break;
       default:
+        if (type in INJECTED_TYPES) {
+          const injectedType = INJECTED_TYPES[type];
+          let customDisplayObject;
+          if (typeof injectedType === "function") {
+            customDisplayObject = injectedType;
+          } else if (typeof injectedType.customDisplayObject === "function") {
+            customDisplayObject = injectedType.customDisplayObject;
+          }
+
+          invariant(
+            customDisplayObject,
+            'Invalid Component injected to ReactPixiFiber: "%s".',
+            type
+          );
+
+          instance = customDisplayObject(props);
+
+          if (typeof injectedType.customApplyProps === "function") {
+            instance._customApplyProps = injectedType.customApplyProps.bind({
+              applyDisplayObjectProps: defaultApplyProps.bind(null, instance)
+            });
+          }
+          if (typeof injectedType.customDidAttach === "function") {
+            instance._customDidAttach = injectedType.customDidAttach;
```

Comment:
As [opposed to `react-pixi`](https://github.com/Izzimach/react-pixi/blob/a25196251a13ed9bb116a8576d93e9fceac2a14c/src/ReactPIXI.js#L297) I'd rather flag this somehow as internal method.